docs/source/README_full.md

```diff
@@ -68,61 +66,53 @@ device = wolk.Device(
 )
 
 # Provide implementation of a way to read actuator status
-class ActuatorStatusProviderImpl(wolk.ActuatorStatusProvider):
-    def get_actuator_status(self, reference):
-        if reference == "SW":
-            return wolk.ActuatorState.READY, switch.value
-        elif reference == "SL":
-            return wolk.ActuatorState.READY, slider.value
+def get_actuator_status(reference):
```

Comment:
Renamed to be more intuitive

Reply:
For me it isn't, provider is the same it one word. But it is also isn't important because we made a decision to name it with "ActuatorStatusProvider" in all connectors, so there is two options. 1 to stay like it was, 2. you can propose changing names in WolkConnects at all and we can discuss about it. But to change it by you own isn't right.

Reply:
I'll rename them to `actuator_status_provider`, `actuation_handler`, `configuration_provider` and `configuration_handler` to stay inline with the rest.